components/loginform.js

```diff
@@ -33,8 +39,21 @@ class LoginForm extends Component {
             switchTwoClicked={() => this.updateLoginType('teacher')}
           />
         </div>
+        {/*TODO SETUP APPROPRIATE ACTION CALL FOR SIGNIN OR SIGNUP*/}
         <form action='api/user/login' method='post'>
           <div className='panel'>
+            {this.state.register && 
```

Comment:
trailing space not allowed.

you can run `npm run lint` and `npm run fix` to auto lint